.github/workflows/perf.yml

```diff
@@ -0,0 +1,26 @@
+name: DiskANN Nightly Performance Metrics
+on:
+  schedule:
+    - cron: "41 14 * * *"  # 14:41 UTC, 7:41 PDT, 8:41 PST, 08:11 IST
+jobs:
+  perf-test:
+    name: Run Perf Test from main
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 1
+      - name: Build Perf Container
+        run: |
+          docker build --build-arg GIT_COMMIT_ISH="$GITHUB_SHA" -t perf -f scripts/perf/Dockerfile scripts
```

Comment:
We could rely on "HEAD", but there's a subtle inconsistency error if someone manages to push a new push to main sometime between this job *starting* to run and the docker build step actually pulling at this commit.

So we're specifically going with the commit-that-was-at-the-start-of-the-job-schedule.  If GIT_COMMIT_ISH is not set, we default to HEAD.